Tests/TuistGeneratorTests/Generator/ProjectGroupsTests.swift

```diff
@@ -70,6 +71,12 @@ final class ProjectGroupsTests: XCTestCase {
         XCTAssertNotNil(main.group(named: "Target"))
         XCTAssertNil(main.group(named: "Target")?.path)
         XCTAssertEqual(main.group(named: "Target")?.sourceTree, .group)
+        
+        let expectedTextSettings = textSettings()
+        XCTAssertEqual(main.usesTabs, expectedTextSettings.usesTabs)
+        XCTAssertEqual(main.indentWidth, expectedTextSettings.indentWidth)
+        XCTAssertEqual(main.tabWidth, expectedTextSettings.tabWidth)
+        XCTAssertEqual(main.wrapsLines, expectedTextSettings.wrapsLines)
```

Comment:
worth another test verifying not setting the options translates to it not being set on the main group